scripts/reborn_webui_v2_live_qa/run_live_qa.py

```
@@ -4059,70 +4687,1514 @@ async def run_gated(ctx: LiveQaContext) -> ProbeResult:
+        personal_token = _require_slack_personal_token(ctx)
+        channel_id = _require_seeded_dm_channel(ctx)
+        counterpart = await _slack_dm_counterpart(personal_token, channel_id)
+        if not counterpart.get("ok"):
+            raise AssertionError(
+                "probe precondition failed: could not resolve the seeded DM "
+                f"counterpart: {counterpart.get('error')!r}"
+            )
+        display_name = str(counterpart.get("display_name") or "")
+        own_user_id = str(counterpart.get("own_user_id") or "")
+        details["counterpart_display_name"] = display_name
+        chat, reply_text = await _slack_correctness_chat_reply(
```

Ruff (0.15.20)

[warning] Unpacked variable `reply_text` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)